meka/srcs/z80marat/Tables.h

```diff
@@ -360,11 +360,11 @@ static byte CyclesED[256] =
   12,12,15,20, 8,14, 8, 9,12,12,15,20, 8,14, 8, 9, // 40-4F
   12,12,15,20, 8,14, 8, 9,12,12,15,20, 8,14, 8, 9, // 50-5F
   12,12,15,20, 8,14, 8,18,12,12,15,20, 8,14, 8,18, // 60-6F
-  12,12,15,20, 8,14, 8, 4,12,12,15,20, 8,14, 8, 4, // 70-7F
+  12,12,15,20, 8,14, 8, 8,12,12,15,20, 8,14, 8, 8, // 70-7F
   NP,NP,NP,NP,NP,NP,NP,NP,NP,NP,NP,NP,NP,NP,NP,NP, // 80-8F (All NOP)
   NP,NP,NP,NP,NP,NP,NP,NP,NP,NP,NP,NP,NP,NP,NP,NP, // 90-9F (All NOP)
   16,16,16,16,NP,NP,NP,NP,16,16,16,16,NP,NP,NP,NP, // A0-AF
-   0, 0, 0, 0,NP,NP,NP,NP, 0, 0, 0, 0,NP,NP,NP,NP, // B0-BF
+  16,16,16,16,NP,NP,NP,NP,16,16,16,16,NP,NP,NP,NP, // B0-BF

```

Comment:
LDIR, CPIR, INIR, OTIR, 
 LDDR, CPDR, INDR, OTDR, 

I don't think this is correct. Meka/Marat version process those in batch, for LDIR/CPIR etc. it removes 16+5 per loop and 16 for last one in the instruction itself.